avocado/utils/pmem.py

```diff
@@ -129,7 +129,7 @@ def get_slot_count(self, region):
     def is_region_legacy(region):
         """
         Check whether we have label index namespace. If legacy we can't create
-        new namespaces.
```

Comment:
I'd argue that it's more natural to keep `namespace` than and add it to the ignore list.

Reply:
I imagined this PR was going to have a comment on this specific term. Looking at it now, namespace makes more sense and is a valid word in the computer science world. I'll change it back and add it to the ignore list.

Reply:
Thanks, it LGTM.

BTW, your comment made me think that a "computer science world" dictionary would be a nice thing to be able to use for our checks. Don't know if such a thing exists though.